### PR TITLE
Remove LoggingScope CRD

### PR DESCRIPTION
**Pull request by pfmackin**
Remove the LoggingScope CRD from the Verrazzano API.   Remove LoggingScope CR from the workload logic, unit tests, integration tests, e2e tests, and examples.  Internally, logging info is still needed by the workload code so keep a simplified logging  struct for that, still use the name logging scope to avoid additional changes that are not really needed.

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: application-operator/controllers/loggingscope/logging_scope.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+const (
+	configMapAPIVersion = "v1"
+	configMapKind       = "ConfigMap"
+)
+
+// LoggingScope contains information needed for logging
+type LoggingScope struct {
+	// The fluentd image
+	FluentdImage string
```
Comment (shihchang) [inline application-operator/controllers/loggingscope/logging_scope.go:26]: Is this FluentdImage still used by any component?

File: application-operator/controllers/loggingscope/helidon.go [hunk 2]
```diff
@@ -192,7 +192,7 @@ func (h *HelidonHandler) Remove(ctx context.Context, workload vzapi.QualifiedRes
 	_, fluentdFound := searchContainers(deploy.Spec.Template.Spec.Containers)
 	var errors []string
 	if fluentdFound {
-		err := h.deleteFluentdConfigMap(ctx, scope.GetNamespace(), workload.Name)
+		err := h.deleteFluentdConfigMap(ctx, scope.SecretNamespace, workload.Name)
```
Comment (mgianatagh) [inline application-operator/controllers/loggingscope/helidon.go:195]: I noticed that sometimes you replaced scope.GetNamespace() with workload.Namespace.  Can either value be used?

File: application-operator/controllers/loggingscope/logging_scope.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2021, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package loggingscope
+
+import (
+	"context"
```
Comment (mgianatagh) [inline application-operator/controllers/loggingscope/logging_scope.go:7]: the order of imports should be updated

File: application-operator/controllers/cohworkload/coherenceworkload_controller.go [hunk 4]
```diff
@@ -326,12 +328,15 @@ func (r *Reconciler) addLogging(ctx context.Context, log logr.Logger, workload *
 		}
 	}
 
-	loggingScope, err := loggingscope.FetchLoggingScopeFromWorkloadLabels(ctx, r.Client, log, workload.Namespace, workload.Labels, existingFluentdImage)
+	// if we're running in a managed cluster, use the multicluster ES URL and secret, and if we're
+	// not the fields will be empty and we will set these fields to defaults below
+	elasticSearchDetails := clusters.FetchManagedClusterElasticSearchDetails(ctx, r.Client)
```
Comment (desagar) [inline application-operator/controllers/cohworkload/coherenceworkload_controller.go:333]: what about other workload types besides weblogic and coherence e.g. helidonworkload? Does that not need the same change?

**Review by mgianatagh — APPROVED** [review]

**Pull request merged**